# Basic Statistics Case Study

# Business Problem-2

## BACKGROUND:

When an order is placed by a customer of a small manufacturing company, a price quote must be developed for that order. Because each order is unique, quotes must be established on an order-by-order basis by a pricing expert. The price quote process is laborintensive, as prices depend on many factors such as the part number, customer, geographic location, market, and order volume. The sales department manager is concerned that the pricing process is too complex, and that there might be too much variability in the quoted prices. An improvement team is tasked with studying and improving the pricing process. After interviewing experts to develop a better understanding of the current process, the team designed a study to determine if there is variability between pricing experts. That is, do different pricing experts provide different price quotes? Two randomly selected pricing experts, Mary and Barry, were asked to independently provide prices for twelve randomly selected orders. Each
expert provided one price for each of the twelve orders.

## BUSINESS PROBLEM:

We would like to assess if there is any difference in the average
price quotes provided by Mary and Barry.

## DATA AVAILABLE:

The data set contains the order number, 1 through 12, and the price quotes by Mary
and Barry for each order. Each row in the data set is the same order. Thus, Mary and
Barry produced quotes for the same orders.

### Import Libraries

In [1]:
import numpy as np
import pandas as pd
import matplotlib.pyplot as plt
import scipy.stats as stats
import seaborn as sns
from scipy import stats

%matplotlib inline

In [2]:
price_data= pd.read_csv('Price_Quotes.csv')
price_data

Order_Number  Barry_Price  Mary_Price
0              1          126         114
1              2          110         118
2              3          138         114
3              4          142         111
4              5          146         129
5              6          136         119
6              7           94          97
7              8          103         104
8              9          140         127
9             10          152         133
10            11          108         103
11            12           97         108

In [3]:
def check_normality(data):
    test_stat_normality, p_value_normality=stats.shapiro(data)
    print("p value:%.4f" % p_value_normality)
    if p_value_normality <0.05:
        print("Reject null hypothesis >> The data is not normally distributed")
    else:
        print("Fail to reject null hypothesis >> The data is normally distributed")  

In [4]:
def check_variance_homogeneity(group1, group2):
    test_stat_var, p_value_var= stats.levene(group1,group2)
    print("p value:%.4f" % p_value_var)
    if p_value_var <0.05:
        print("Reject null hypothesis >> The variances of the samples are different.")
    else:
        print("Fail to reject null hypothesis >> The variances of the samples are same.")

### 1. Defining Hypothesis

We would like to assess if there is any difference in the average price quotes provided by Mary and Barry.

#### H0: There is no difference in the average price quotes provided by Mary and Barry.
#### H1: There is difference in the average price quotes provided by Mary and Barry.

The performance of the methods by using a 0.05 significance level. the hypothesis testing to check whether there is a difference between the performance of the methods by using a 0.05 significance level.

### 2. Assumption Check

    H₀: The data is normally distributed.
    H₁: The data is not normally distributed.

    H₀: The variances of the samples are the same.
    H₁: The variances of the samples are different.

In [5]:
check_normality(price_data.Mary_Price)
check_normality(price_data.Barry_Price)

p value:0.9288
Fail to reject null hypothesis >> The data is normally distributed
p value:0.1644
Fail to reject null hypothesis >> The data is normally distributed


In [6]:
check_variance_homogeneity(price_data.Mary_Price,price_data.Barry_Price)

p value:0.0190
Reject null hypothesis >> The variances of the samples are different.


### 3. Selecting the Proper Test

In [7]:
statsat, p_value = stats.ttest_rel(price_data.Mary_Price,price_data.Barry_Price)
print("p value:%.6f" % p_value)
if p_value <0.05:
    print("Reject null hypothesis")
else:
    print("Fail to reject null hypothesis")

p value:0.028406
Reject null hypothesis


### 4. Decision and Conclusion

we observe a p-value of 0.0284.This confirms what we learned with the confidence interval for the difference there is a significant difference in average price quotes provided by Mary and Barry. 